app/api/server/v1/misc.js

```diff
@@ -88,6 +88,10 @@ API.v1.addRoute('shield.svg', { authRequired: false, rateLimiterOptions: { numRe
 				text = `#${ channel }`;
 				break;
 			case 'user':
+				const loggedUser = this.getLoggedInUser();
+				if (settings.get('API_Shield_user_require_auth') && !loggedUser) {
+					return API.v1.failure('You must be logged in to do this.');
+				}
```

Comment:
I just realised you don't need to call `getLoggedInUser()` if `API_Shield_user_require_auth` is `false` 😬 
```suggestion
				if (settings.get('API_Shield_user_require_auth') && !this.getLoggedInUser()) {
					return API.v1.failure('You must be logged in to do this.');
				}
```